benchmark_runner/common/prometheus/metrics-default.yaml

```diff
@@ -1,267 +1,10 @@
+  metricName: containerCPU-benchmark-runner
 
-# Container & pod metrics
-- query: (sum(container_memory_rss{name!="",container!="POD",namespace=~"openshift-(etcd|oauth-apiserver|.*apiserver|ovn-kubernetes|sdn|ingress|authentication|.*controller-manager|.*scheduler)"}) by (container, pod, namespace, node) and on (node) kube_node_role{role="master"}) > 0
-  metricName: containerMemory-Masters
+- query: sum(container_memory_rss{namespace="benchmark-runner"} and on (node) kube_node_role{role="worker"}) by (node) > 0
+  metricName: containerMemory-benchmark-runner
 
-- query: (sum(irate(container_cpu_usage_seconds_total{name!="",container!="POD",namespace=~"openshift-(etcd|oauth-apiserver|sdn|ovn-kubernetes|.*apiserver|authentication|.*controller-manager|.*scheduler)"}[2m]) * 100) by (container, pod, namespace, node) and on (node) kube_node_role{role="master"}) > 0
-  metricName: containerCPU-Masters
 
-- query: (sum(irate(container_cpu_usage_seconds_total{pod!="",container="prometheus",namespace="openshift-monitoring"}[2m]) * 100) by (container, pod, namespace, node) and on (node) kube_node_role{role="infra"}) > 0
-  metricName: containerCPU-prometheus
-
-- query: (avg(irate(container_cpu_usage_seconds_total{name!="",container!="POD",namespace=~"openshift-(sdn|ovn-kubernetes|ingress)"}[2m]) * 100 and on (node) kube_node_role{role="worker"}) by (namespace, container)) > 0
-  metricName: containerCPU-AggregatedWorkers
-
-- query: (avg(irate(container_cpu_usage_seconds_total{name!="",container!="POD",namespace=~"openshift-(sdn|ovn-kubernetes|ingress|monitoring|image-registry|logging)"}[2m]) * 100 and on (node) kube_node_role{role="infra"}) by (namespace, container)) > 0
-  metricName: containerCPU-AggregatedInfra
-
-- query: (sum(container_memory_rss{pod!="",namespace="openshift-monitoring",name!="",container="prometheus"}) by (container, pod, namespace, node) and on (node) kube_node_role{role="infra"}) > 0
-  metricName: containerMemory-prometheus
-
-- query: avg(container_memory_rss{name!="",container!="POD",namespace=~"openshift-(sdn|ovn-kubernetes|ingress)"} and on (node) kube_node_role{role="worker"}) by (container, namespace)
-  metricName: containerMemory-AggregatedWorkers
-
-- query: avg(container_memory_rss{name!="",container!="POD",namespace=~"openshift-(sdn|ovn-kubernetes|ingress|monitoring|image-registry|logging)"} and on (node) kube_node_role{role="infra"}) by (container, namespace)
-  metricName: containerMemory-AggregatedInfra
-
-- query: (sum(container_memory_rss{pod!="",namespace=~"{{ namespace_re }}",name!=""}) by (node)) > 0
-  metricName: containerMemoryRSS-clusterbuster
-
-- query: (sum(container_memory_working_set_bytes{pod!="",namespace=~"{{ namespace_re }}",name!=""}) by (node)) > 0
-  metricName: containerMemoryWorkingSet-clusterbuster
-
-- query: (sum(irate(container_cpu_usage_seconds_total{container!="",namespace=~"{{ namespace_re }}"}[2m])) by (node)) > 0
-  metricName: containerCPU-clusterbuster
```

Comment:
This needs to be fixed (rather than the templated namespace_re, just use clusterbuster), but it's useful for the clusterbuster tests.